pages/index.tsx

```diff
@@ -112,13 +113,13 @@ const App = ({ name, numberOfTejass }) => {
         <b>TEJ</b>
         {name}
       </Title>
-      <img alt={`Tejas ${currentTejas}`} src={`/static/tejass/${currentTejas}.png`} />
+      <img alt={`Tejas ${currentTejas} ${attributions[currentTejas] || ""}`} src={`/static/tejass/${currentTejas}.png`} />
```

Comment:
We don't need this.

Reply:
Cool. I'm trying too hard to be a nice guy. I should tell you about that time I expected to meet the lawyers.